compiler/src/lowering/mappers/module.rs

```diff
@@ -15,7 +18,46 @@ pub fn module(context: &Context, module: fe::Module) -> fe::Module {
             fe::ModuleStmt::SimpleImport { .. } => stmt,
             fe::ModuleStmt::ContractDef { .. } => contracts::contract_def(context, stmt),
         })
+        .collect::<Vec<_>>();
+
+    let attributes = context.get_module().expect("missing attributes");
+
+    let tuple_defs = attributes
```

Comment:
`tuple_defs` might be a slightly confusing name here as the bindings effectively holds struct definitions.

Suggestions:

- `struct_defs`
- `mapped_defs`
- `ex_tuple_defs`
- `new_struct_defs`

Or whatever hints best about the converted defs :sweat_smile: